jubilant/secrettypes.py

```diff
@@ -0,0 +1,143 @@
+"""Dataclasses that contain parsed output from juju secrets."""
+
+from __future__ import annotations
+
+import dataclasses
+import datetime
+import enum
+from typing import Dict, Generic, TypedDict, TypeVar
+
+from typing_extensions import Required
+
+
+class SecretRotateCadence(enum.Enum):
```

Comment:
Could these type names all drop the `Secret` prefix, given they're in the `secrettypes` module? Also, this is called `SecretRotate` in ops, so I think we can drop the `Cadence` suffix too.
```suggestion
class Rotate(enum.Enum):
```

But obviously not for `Secret` itself, and `SecretURI` is imported to the top level, so  not that one either.

Reply:
done